elastic/apm-server
# Add mapping for cgroup metrics.

## Motivation/summary

Define mapping for `system.process.cgroup.*` metrics sent by APM agents.

## Checklist

~- [ ] I have signed the [Contributor License Agreement](https://www.elastic.co/contributor-agreement/).~

I have considered changes for:
- [x] documentation (generated)
~- [ ] logging (add log lines, choose appropriate log selector, etc.)~
~- [ ] metrics and monitoring (create issue for Kibana team to add metrics to visualizations, e.g. [Kibana#44001](https://github.com/elastic/kibana/issues/44001))~
~- [ ] telemetry~
~- [ ] Elasticsearch Service (https://cloud.elastic.co)~
~- [ ] Elastic Cloud Enterprise (https://www.elastic.co/products/ece)~
~- [ ] Elastic Cloud on Kubernetes (https://www.elastic.co/elastic-cloud-kubernetes)~

## How to test these changes
start `apm-server -e` to set up the templates and check the created mapping;
ingest some [`cgroup` metrics](https://github.com/elastic/apm/blob/master/specs/agents/metrics.md#cgroup-metrics) 

## Related issues
closes #4163
